metrics/README

```diff
@@ -0,0 +1,7 @@
+OpenCensus Metrics
```

Comment:
I think this file should be README.md, for consistency with the other readmes.

Reply:
Good catch, done.